doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1003 @@
+
+* :ref:`lwm2m_client` sample:
+
+  * Added:
+
+    * Support for nRF7002 EK shield and Wi-Fi based location.
+    * Location events and event handlers.
+
+  * Updated:
+
+    * The sensor module has been simplified.
+      It does not use application events, filtering, or configurable periods anymore.
+    * Replaced deprecated LwM2M API calls with calls to new functions.
+    * Enabled LwM2M queue mode and updated documentation accordingly.
+    * Moved configuration options from the :file:`overlay-queue.conf` file to the default configuration file :file:`prj.conf`.
+    * Removed the :file:`overlay-queue.conf` file.
+    * Enabled the :kconfig:option:`CONFIG_LTE_LC_TAU_PRE_WARNING_NOTIFICATIONS` Kconfig option.
+
+* :ref:`http_application_update_sample` sample:
+
+  * Added support for the :ref:`liblwm2m_carrier_readme` library.
+
+* :ref:`nrf_cloud_mqtt_multi_service` sample:
+
+  * Added:
+
+    * MCUboot child image files to properly access external flash on newer nRF9160 DK versions.
+    * An :file:`overlay_mcuboot_ext_flash.conf` file to enable MCUboot to use external flash.
+    * Sending an alert to the cloud on boot and when a temperature limit is exceeded.
+
+  * Updated the sample to use a partition in external flash for full modem FOTA updates.
+
+* :ref:`nrf_cloud_rest_device_message` sample:
+
+  * Added sending an alert to nRF Cloud on boot.
+
+* Removed:
+
+  * Multicell location sample because of the deprecation of the Multicell location library.
+    Relevant functionality is available through the :ref:`lib_location` library.
+  * nRF9160: Simple MQTT sample.
+    This is now replaced by a new :ref:`mqtt_sample` sample that supports Wi-Fi and LTE connectivity.
+
+* :ref:`nrf_cloud_rest_fota` sample:
+
+  * Updated:
+
+    * Device status information, including FOTA enablement, is now sent to nRF Cloud when the device connects.
+    * Removed user prompt and button press handling for FOTA enablement.
+    * The sample now uses a partition in external flash for full modem FOTA updates.
+
+* :ref:`azure_fota_sample` sample:
+
+  * Updated the sample to use the logging subsystem for console output.
+
+* :ref:`azure_iot_hub` sample:
+
+  * Updated the sample to use the logging subsystem for console output.
+
+* :ref:`aws_iot` sample:
+
+  * Updated the sample to use the logging subsystem for console output.
+
+Thread samples
+--------------
+
+* Updated the overlay structure:
+
+    * The :file:`overlay-rtt.conf` file was removed from all samples.
+    * The :file:`overlay-log.conf` file now uses RTT backend by default.
+    * Logs removed from default configuration (moved to :file:`overlay-logging.conf`).
+    * Asserts removed from default configuration (moved to :file:`overlay-debug.conf`).
+
+* :ref:`ot_cli_sample`:
+
+  * Added the :file:`overlay-low_power.conf` and :file:`low_power.overlay` files to facilitate power consumption measurements.
```

Comment:
Is this entry specific to this sample? In the changelog it appears to be a common entry.

Reply:
The original text says: 
 * :file:`overlay-low_power.conf` and :file:`low_power.overlay` **to the CLI sample** to facilitate power consumption measurements.
 , so that's why I made this format change.